docs/api/alerts/create.asciidoc

```diff
@@ -0,0 +1,161 @@
+[[alerts-api-create]]
+=== Create alert API
+++++
+<titleabbrev>Create alert</titleabbrev>
+++++
+
+Create {kib} alerts.
+
+[[alerts-api-create-request]]
+==== Request
+
+`POST <kibana host>:<port>/api/alerts/alert`
+
+[[alerts-api-create-request-body]]
+==== Request body
+
+`name`::
+  (Required, string) A name to reference and search in the future.
+
+`tags`::
+  (Optional, string array) A list of keywords to reference and search in the future.
+
+`alertTypeId`::
+  (Required, string) The id value of the alert type you want to call when the alert is scheduled to execute.
+
+`schedule`::
+  (Required, object) The schedule specifying when this alert should be run, using one of the available schedule formats specified under _Schedule Formats_ below.
+
+`throttle`::
+  (Optional, string) A Duration specifying how often this alert should fire the same actions. This will prevent the alert from sending out the same notification over and over. For example, if an alert with a `schedule` of 1 minute stays in a triggered state for 90 minutes, setting a `throttle` of `10m` or `1h` will prevent it from sending 90 notifications over this period.
+
+`notifyWhen`::
+  (Required, string) Defines the conditions for notifications thottling from the three available option types 'onActionGroupChange', 'onActiveAlert', 'onThrottleInterval'.
+
+`enabled`::
+  (Optional, boolean) Indicate if you want the alert to start executing on an interval basis after it has been created.
+
+`consumer`::
+  (Required, string) A name of the application which owns the alert.
```

Comment:
It might be worth explicitly stating that the `Consumer` and ` Producer` fields have to match the Kibana Feature names, as that dictates the required RBAC privileges.
Otherwise people won't know what values to put here I think.

Reply:
Yes, it makes sense to me.